test/python/compiler/test_assembler.py

```diff
@@ -262,7 +262,7 @@ class TestPulseAssembler(QiskitTestCase):
     """Tests for assembling schedules to qobj."""
```

Comment:
The tests should be duplicated for now for `DeviceSpecification` until deprecation, although tests that use `DeviceSpecification` can just be switched to `PulseSpecification` as you have.